### PR TITLE
[Brownfield] Tests refactoring

### PR DESCRIPTION
**Pull request by annakocheshkova**
Created a sharable test utils module. Renamed tests.

### CODE REVIEW

File: android/app/src/main/java/com/microsoft/codepush/common/utils/FileUtils.java [hunk 1]
```diff
@@ -189,6 +189,7 @@ public static String readFileToString(String filePath) throws IOException {
             Exception e = finalizeResources(
                     Arrays.asList(reader, fileInputStream),
                     "Error closing IO resources when reading file.");
+
```
Comment (sergey-akhalkov) [inline android/app/src/main/java/com/microsoft/codepush/common/utils/FileUtils.java]: empty line - please add a comment after that or remove it.

File: android/app/src/main/java/com/microsoft/codepush/common/utils/PlatformUtils.java [hunk 2]
```diff
@@ -2,14 +2,15 @@
 
 /**
  * Interface describing the methods that should be implemented in platform-specific instances of utils.
+ * It can be implemented via platform-specific singleton.
```
Comment (sergey-akhalkov) [inline android/app/src/main/java/com/microsoft/codepush/common/utils/PlatformUtils.java]: Don't sure if we need this comment, is it some kind of todo?

Comment (annakocheshkova) [reply android/app/src/main/java/com/microsoft/codepush/common/utils/PlatformUtils.java]: @sergey-akhalkov This is just a note on interface use case

File: android/app/src/main/java/com/microsoft/codepush/common/utils/FileUtils.java (hunk 1 above)
Comment (annakocheshkova) [reply android/app/src/main/java/com/microsoft/codepush/common/utils/FileUtils.java]: Removed.

**Pull request merged**